ecoli/analysis/multivariant/cell_mass.py

```diff
@@ -0,0 +1,162 @@
+"""
+Plot absolue / normalized cell mass over time for multivariant simulation in vEcoli, and:
+1. each variant has its own plot;
+2. at each subplot, time is divided by generation id;
+
+It can also be used at multigeneration analysis.
+"""
+
+import os
+from typing import Any
+import altair as alt
+import polars as pl
+import pandas as pd
+from duckdb import DuckDBPyConnection
+
+
+def plot(
+    params: dict[str, Any],
+    conn: DuckDBPyConnection,
+    history_sql: str,
+    config_sql: str,
+    success_sql: str,
+    sim_data_dict: dict[str, dict[int, str]],
+    validation_data_paths: list[str],
+    outdir: str,
+    variant_metadata: dict[str, dict[int, Any]],
+    variant_names: dict[str, str],
+):
+    # Load data with required columns
+    required_columns = [
+        "time",
+        "variant",
```

Comment:
It's kinda impossible to properly see anything with all the seeds lumped together. Include `lineage_seed` here.
```suggestion
        "variant",
        "lineage_seed",
```